src/modules/translation/utils.ts

```diff
@@ -51,7 +51,7 @@ export function getCurrentLocale() {
 }
 
 export function t(id: string, values?: any) {
-  return currentLocale.formatMessage({ id }, values)
+  return currentLocale ? currentLocale.formatMessage({ id }, values) : ''
```

Comment:
The problem here was that the tests would blow up because the TranslationProvider is not present. I made it this way so the tests could run, in practice this shouldn't happen